data/bedrock/latest/proto.yml

```diff
@@ -4217,3 +4217,45 @@ packet_server_post_move:
 packet_clientbound_close_form:
    !id: 0x136
    !bound: client
+
+# ServerBoundLoadingScreen is sent by the client to tell the server about the state of the loading
+# screen that the client is currently displaying.
+packet_serverbound_loading_screen:
+   !id: 0x138
+   !bound: server
+   # The type of the loading screen event.
+   type: zigzag32
+   loading_screen_id?: varint
+
+# JigsawStructureData is sent by the server to let the client know all the rules for jigsaw structures.
+packet_jigsaw_structure_data:
+   !id: 0x139
+   !bound: client
+   # StructureData is a network NBT serialised compound of all the jigsaw structure rules defined
+   # on the server.
+   structure_data: string
```

Comment:
Is this not supposed to be NBT?

Reply:
I rechecked it and you would be correct. I've fixed the issue.